### PR TITLE
Add new txt documents

### PR DESCRIPTION
**Pull request by nasli**

### CODE REVIEW

File: Owncloud iOs Client/Files/Edit/EditFileViewController.m [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+
+/*
+ Copyright (C) 2016, ownCloud, Inc.
+ This code is covered by the GNU Public License Version 3.
+ For distribution utilizing Apple mechanisms please see https://owncloud.org/contribute/iOS-license-exception/
+ You should have received a copy of this license
+ along with this program. If not, see <http://www.gnu.org/licenses/gpl-3.0.en.html>.
+ */
+
+#import "EditFileViewController.h"
```
Comment (mRoRo) [inline Owncloud iOs Client/Files/Edit/EditFileViewController.m]: I think Customization.h, OCCommunication.h and ManageNetworkErrors.h are not used

File: Owncloud iOs Client/Utils/InfoFileUtils.m [hunk 2]
```diff
@@ -227,9 +230,17 @@ + (CustomCellFileAndDirectory *) getTheStatusIconOntheFile: (FileDto *)fileForSe
             }
         } else if (!fileForSetTheStatusIcon.isFavorite) {
             if(fileForSetTheStatusIcon.isNecessaryUpdate || fileForSetTheStatusIcon.isDownload == updating) {
+            //if(fileForSetTheStatusIcon.isNecessaryUpdate) {
                 //File is in updating
+                
+                //TODO:pending update DB propertly after exist the file on DB and remove check here fileSystem
```
Comment (javiergonzper) [inline Owncloud iOs Client/Utils/InfoFileUtils.m]: The TODOs are necessary ?

Comment (nasli) [inline Owncloud iOs Client/Utils/InfoFileUtils.m]: Once we use the commented code yes. Not for now, I removes all comment. ;)

File: Owncloud iOs Client/Files/Edit/EditFileViewController.m (hunk 1 above)
Comment (nasli) [inline Owncloud iOs Client/Files/Edit/EditFileViewController.m]: 👍

File: Owncloud iOs Client/Files/Edit/EditFileViewController.h [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+@property (nonatomic, strong) ManageNetworkErrors *manageNetworkErrors;
+
+@property(nonatomic, strong) NSMutableArray *currentDirectoryArray;
+@property(nonatomic, strong) FileDto *currentFileDto;
+@property(nonatomic) UIAlertView *alert;
+
+@property (weak, nonatomic) IBOutlet UITextField *titleTextField;
+@property (weak, nonatomic) IBOutlet UITextView *bodyTextView;
+
+- (id)initWithFileDto:(FileDto *)fileDto;
```
Comment (mRoRo) [inline Owncloud iOs Client/Files/Edit/EditFileViewController.h]: Please check if ManageNetworkErrorsDelegate, manageNetworkErrors and alert are necessary/used

File: Owncloud iOs Client/Files/Edit/EditFileViewController.m [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+#pragma mark - Upload text file
+
+- (void) sendTextFileToUploadsByTempLocalPath:(NSString *)tempLocalPath andFileName:(NSString *)fileName {
+   
+    AppDelegate *app = (AppDelegate *)[[UIApplication sharedApplication]delegate];
+
+    NSString *fullRemotePath = [NSString stringWithFormat:@"%@",[UtilsUrls getFullRemoteServerFilePathByFile:self.currentFileDto andUser:app.activeUser]];
+    
+    long long fileLength = [[[[NSFileManager defaultManager] attributesOfItemAtPath:tempLocalPath error:nil] valueForKey:NSFileSize] unsignedLongLongValue];
+
```
Comment (mRoRo) [inline Owncloud iOs Client/Files/Edit/EditFileViewController.m]: Maybe adding here unsigned to the long long variable

File: Owncloud iOs Client/Files/Edit/EditFileViewController.m [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+        upload.isLastUploadFileOfThisArray = YES;
+        upload.status = pendingToBeCheck;
+        upload.chunksLength = k_lenght_chunk;
+        upload.isNotNecessaryCheckIfExist = NO;
+        upload.isInternalUpload = NO;
+        upload.taskIdentifier = 0;
+        
+        [ManageUploadsDB insertUpload:upload];
+       // [app relaunchUploadsFailedForced];
+        [app initUploadsOffline];
```
Comment (mRoRo) [inline Owncloud iOs Client/Files/Edit/EditFileViewController.m]: delete comment if not necessary

File: Owncloud iOs Client/Network/Uploads/ManageUploadRequest.m [hunk 6]
```diff
@@ -33,6 +33,7 @@
 #import "OCURLSessionManager.h"
 #import "ManageAppSettingsDB.h"
 #import "UtilsCookies.h"
+#import "UtilsFileSystem.h"
```
Comment (mRoRo) [inline Owncloud iOs Client/Network/Uploads/ManageUploadRequest.m]: It seems OCURLSessionManager.h and UtilsFileSystem.h are not used

File: Owncloud iOs Client/Utils/UtilsFileSystem.m [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+ Copyright (C) 2016, ownCloud, Inc.
+ This code is covered by the GNU Public License Version 3.
+ For distribution utilizing Apple mechanisms please see https://owncloud.org/contribute/iOS-license-exception/
+ You should have received a copy of this license
+ along with this program. If not, see <http://www.gnu.org/licenses/gpl-3.0.en.html>.
+ */
+
+#import "UtilsFileSystem.h"
+#import "UtilsUrls.h"
+#import "UploadsOfflineDto.h"
```
Comment (mRoRo) [inline Owncloud iOs Client/Utils/UtilsFileSystem.m]: Please check if UploadsOfflineDto is necesssary

File: Owncloud iOs Client/Files/Edit/EditFileViewController.h (hunk 3 above)
Comment (nasli) [inline Owncloud iOs Client/Files/Edit/EditFileViewController.h]: 👍 Removed

File: Owncloud iOs Client/Files/Edit/EditFileViewController.m (hunk 5 above)
Comment (nasli) [inline Owncloud iOs Client/Files/Edit/EditFileViewController.m]: 👍 Removed

File: Owncloud iOs Client/Network/Uploads/ManageUploadRequest.m (hunk 6 above)
Comment (nasli) [inline Owncloud iOs Client/Network/Uploads/ManageUploadRequest.m]: SessionManager is necessary, the other one removed 👍

File: Owncloud iOs Client/Utils/UtilsFileSystem.m (hunk 7 above)
Comment (nasli) [inline Owncloud iOs Client/Utils/UtilsFileSystem.m]: 👍 Removed

**Pull request merged**